dist/server/PublishRouter.js

```diff
@@ -0,0 +1,61 @@
+const PublishJob = require('./PublishJob.js');
+const express = require('express');
+const uuid = require('uuid');
+const bodyParser = require('body-parser');
+
+module.exports = function(port, rootUrl, unifile) {
+
+  const router = express.Router();
+
+  // **
+  // publication tasks
+  router.post('/tasks/:task', bodyParser.json(), (req, res, next) => {
+    // init the session (shouldn't it be done by express??
+    req.session.sessionID = req.session.sessionID || uuid.v4();
```

Comment:
Yes it should. Did you encounter a case where the session was not set?

Reply:
I guess I did
I'll remove it and see what happens since we updated express